.gitignore

```diff
@@ -81,3 +81,6 @@ Temporary Items
 *.orig
 *.rej
 *.bak
+
+# Ignore positions file (#3518)
+positions.js
```

Comment:
can you please change to `js/positions.js` because there is another file with the same name in this repo `tests/configs/modules/positions.js` ...

Reply:
Oh, sorry. I just fixed it.